src/python/examples/inference_example.py

```diff
@@ -0,0 +1,45 @@
+import pathlib
+
+import numpy as np
+import tritonclient.http as httpclient
+import tritonserver
+from tritonfrontend import KServeHttp
+
+model_path = f"{pathlib.Path(__file__).parent.resolve()}/../test/test_model_repository"
+server_options = tritonserver.Options(
+    server_id="ExampleServer",
+    model_repository=model_path,
+    log_error=True,
+    log_warn=True,
+    log_info=True,
+)
+server = tritonserver.Server(server_options).start()
+
+http_options = KServeHttp.Options()
+http_service = KServeHttp.Server(server, http_options)
+http_service.start()
+
+model_name = "identity"
+url = "0.0.0.0:8000"
```

Comment:
Should this be `localhost` instead of `0.0.0.0` from client perspective?

Reply:
Nice catch! I defaulted to `0.0.0.0` instead of `localhost` because the frontends don't like `localhost`.
But from a client-facing perspective it should be `localhost`. Changed to `url = localhost:8000`